pkg/opt/opt.go

```diff
@@ -63,25 +65,20 @@ func (opts *GitOptions) Validate() error {
 	return nil
 }
 
-const DEFAULT_REGISTRY = "docker.io"
 const DEFAULT_OCI_TAG = "latest"
 
-func GetOCIReg() string {
-	return DEFAULT_REGISTRY
-}
-
 func GetDefaultOCITag() string {
 	return DEFAULT_OCI_TAG
 }
 
 type OciOptions struct {
```

Comment:
Add some comments for `OciOptions`: such as {reg}/{repo}/{pkg}/{tag}

Reply:
fixed.